grpc.proto

```diff
@@ -67,6 +68,10 @@ message TrafficOut {
     bool is_broadcast = 3;
 }
 
+message Abort {
```

Comment:
Do not define a new message type for `abort`.  Instead, `abort` can just be a `bool` directly.

Reply:
Defined `Abort` as a struct because I suppose we will have more data associated with timeouts in the future. I guess we can change it back into a struct in the future if needed.